File: extra/house_price.r
# Sample dataset
data <- data.frame(
  Area = c(1000,1500,2000,2500,3000),
  Bedrooms = c(2,3,3,4,4),
  Age = c(10,5,3,2,1),
  Price = c(200000,300000,400000,500000,600000)
)

# Train model
model <- lm(Price ~ Area + Bedrooms + Age, data=data)

# View model summary
summary(model)

# Predict new house price
new_house <- data.frame(Area=2200, Bedrooms=3, Age=5)
predict(model, new_house)

File: extra/house_price.py
#Implement multi linear regression using python or r to predict the price of house 

import pandas as pd
from sklearn.model_selection import train_test_split
from sklearn.linear_model import LinearRegression

# Example dataset
data = {
    'Area': [1000, 1500, 2000, 2500, 3000],
    'Bedrooms': [2, 3, 3, 4, 4],
    'Age': [10, 5, 3, 2, 1],
    'Price': [200000, 300000, 400000, 500000, 600000]
}

df = pd.DataFrame(data)

# Features (independent variables)
X = df[['Area', 'Bedrooms', 'Age']]

# Target variable
y = df['Price']

# Split dataset
X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.2)

# Create model
model = LinearRegression()

# Train model
model.fit(X_train, y_train)

# Predict price
prediction = model.predict([[2200, 3, 5]])

print("Predicted House Price:", prediction[0])

# Model coefficients
print("Coefficients:", model.coef_)
print("Intercept:", model.intercept_)
